VolumeLogic/Application.py

```diff
@@ -20,11 +21,12 @@ def name(self, name):
 
     @property
     def volume(self):
-        return self.volume
+        return self.os_connection.getProcessVolume(self.name)
```

Comment:
Are you setting `os_connection `anywhere? I see it below but shouldn't it be part of this `__init__ `instead of relying on it being set elsewhere?

Reply:
https://github.com/Free-Brick-Studio/Bricxer/pull/8/files#diff-d24cf91b089c3027742d878bd22085fed0925af192e1ca860bdeeade3104c81bR16

I made it a class variable since each instance will be hitting the same OS connector. Then had it set once in the _init_ of VolumeMixer. I can add getters and setters if you would rather